spring-cloud-dataflow-server/src/test/java/org/springframework/cloud/dataflow/integration/test/oauth/DataflowOAuthIT.java

```diff
@@ -63,26 +63,19 @@ public void testSecuredSetup() throws Exception {
 					boolean ok = response.contains("\"authenticated\":true") && response.contains("\"username\":\"janne\"");
 					log.info("Check for oauth {}", ok);
 					if (ok) {
-						try {
-							String version = JsonPath.parse(response).read("$.versionInfo.core.version");
-							log.info("Version=[{}]", version);
-							String api = "tasks/executions";
-							if (VersionUtils.isDataFlowServerVersionGreaterThanOrEqualToRequiredVersion(
-									VersionUtils.getThreePartVersion(version), "2.11.3")) {
-								api = "tasks/thinexecutions";
-							}
-							log.info("Checking {}", api);
-							cmdResult = execInToolsContainer("curl", "-u", "janne:janne", "http://dataflow:9393/" + api);
-							response = cmdResult.getStdout();
-							log.debug("Response is {}", response);
-							ok = !JsonPath.parse(response).read("$._links.self.href", String.class).isEmpty();
-
-							// TODO add checks for new endpoints to check security
-						}
-						catch (Throwable x) {
-							log.error("Exception:" + x, x);
-							ok = false;
+						String version = JsonPath.parse(response).read("$.versionInfo.core.version");
+						log.info("Version=[{}]", version);
+						String api = "tasks/executions";
+						if (VersionUtils.isDataFlowServerVersionGreaterThanOrEqualToRequiredVersion(
+								VersionUtils.getThreePartVersion(version), "2.11.3")) {
+							api = "tasks/thinexecutions";
 						}
+						log.info("Checking {}", api);
+						cmdResult = execInToolsContainer("curl", "-u", "janne:janne", "http://dataflow:9393/" + api);
+						response = cmdResult.getStdout();
+						log.debug("Response is {}", response);
+						ok = !JsonPath.parse(response).read("$._links.self.href", String.class).isEmpty();
+						// TODO add checks for new endpoints to check security
```

Comment:
I don't think this test is the place to add endpoints to. I still would like to remove this comment.